gr-utils/python/modtool/core/rm.py

```diff
@@ -137,6 +137,7 @@ def _run_subdir(self, path, globs, makefile_vars, cmakeedit_func=None):
                         may contain references to the globbed files
         cmakeedit_func - If the CMakeLists.txt needs special editing, use this
         """
+        from ..cli import cli_input
```

Comment:
needs checking whether `self.cli`; we're running into the same old "should work without interactive terminal if all necessary arguments are given, but doesn't" problems else :)

Reply:
the `if not yes:` should be a `if not yes and self.cli:`. See the `run` method!